erc20-migration/test/zenclaim_test.js

```diff
@@ -93,6 +98,21 @@ describe("ZEND Claim test", function () {
     var pubKey4 = zencashjs.address.privKeyToPubKey(privKey4, false) // generate uncompressed pubKey  
     var zAddr4 = zencashjs.address.pubKeyToAddr(pubKey4);
     TEST4_ZEND_ADDRESS = "0x" + bs58check.decode(zAddr4).toString("hex").slice(4); //remove the chain prefix
+
+    TEST_DIRECT_BASE_ADDRESS = "0x6ebacd4a2a48728e98aAAA101C59f2e0c57fA987";
+    var prefix = '2089'
+    //calculate correspondant zend address and load direct value
+    var directZENDTransferAddress = bs58check.encode(
+      Buffer.from(
+        prefix +
+        createHash('rmd160').update(
+          createHash('sha256').update(
+            Buffer.from(TEST_DIRECT_BASE_ADDRESS.substring(2), 'hex')
+          ).digest()
+        ).digest('hex'),
+      'hex')
+    )
+    TEST_DIRECT_ZEND_ADDRESS = "0x" + bs58check.decode(directZENDTransferAddress).toString("hex").slice(4); //remove the chain prefix
```

Comment:
you are encoding base58 and right after decoding :-)
could be skipped, and just calculate rmd160 + sha256
but it's ok we can leave it to let explain the whole conversions steps